### ISSUE TITLE
Linting no longer works

### ISSUE THREAD

**Issue by ffxsam** [meteor/meteor#6414]
I'm using 1.3 beta 12. I noticed `dburles:eslint` is not working, so I tried the official `jshint` package, which also fails. Both yield the same error when I try to run meteor:

jshint:
```
   While linting files with jshint for the app (Server):
   /tools/isobuild/build-plugin.js:476:14: Object [object Object] has no method
   'getFileOptions'
   at [object Object].error (/tools/isobuild/build-plugin.js:476:14)
   at packages/lintJshint/plugin/lint-jshint.js:109:1
   at Array.forEach (native)
   at reportErrors (packages/lintJshint/plugin/lint-jshint.js:108:1)
   at packages/lintJshint/plugin/lint-jshint.js:102:1
   at Array.forEach (native)
   at JsHintLinter.processFilesForPackage
   (packages/lintJshint/plugin/lint-jshint.js:87:1)
```

eslint:
```
   While linting files with dburles:eslint for the app (Client):
   /tools/isobuild/build-plugin.js:476:14: Object [object Object] has no method
   'getFileOptions'
   at [object Object].error (/tools/isobuild/build-plugin.js:476:14)
   at packages/eslint/plugin/eslint.js:126:1
   at Array.forEach (native)
   at reportErrors (packages/eslint/plugin/eslint.js:125:1)
   at packages/eslint/plugin/eslint.js:119:1
   at Array.forEach (native)
   at EsLintLinter.processFilesForPackage
   (packages/eslint/plugin/eslint.js:104:1)
```

**Comment by pcorey** [comment]
[`east5th:check-checker`](https://github.com/East5th/check-checker/) is also [affected by this](https://github.com/East5th/check-checker/issues/9). Let me know if there's been a change to the API, so I can fix the package for the upcoming release.

**Comment by benjamn** [comment]
Fixed in `1.3-rc.1`, thanks to https://github.com/meteor/meteor/commit/711d3674baf2bdd97a8ef577da298485593a0150.
